openmetadata-ui/src/main/resources/ui/src/utils/AdvancedSearchUtils.tsx

```diff
@@ -136,3 +136,31 @@ export const renderAdvanceSearchButtons: RenderSettings['renderButton'] = (
 
   return <></>;
 };
+
+export const getSearchDropdownLabels = (
+  optionsArray: string[],
+  checked: boolean
+): MenuProps['items'] =>
+  optionsArray.map((option) => ({
+    key: option,
+    label: (
+      <Space className="m-x-sm" data-testid={option} size={6}>
+        <Checkbox checked={checked} data-testid={`${option}-checkbox`} />
+        <Typography.Text
+          ellipsis
+          className="dropdown-option-label"
+          title={option}>
+          {option}
+        </Typography.Text>
+      </Space>
+    ),
+  }));
+
+export const getSelectedOptionLabelString = (selectedOptions: string[]) => {
+  const stringifiedOptions = selectedOptions.join(', ');
```

Comment:
Add check to see wether received param is array or not

Reply:
Added checks for possible failure cases. Can you please review?